packages/core/src/Page/index.js

```diff
@@ -354,10 +354,15 @@ class Page {
   generatePageNavHeadingHtml() {
     let headingHTML = '';
     const headingStack = [];
+    const interpolationRegex = /{{.*}}/;
     Object.keys(this.navigableHeadings).forEach((key) => {
       const currentHeadingLevel = this.navigableHeadings[key].level;
-      const currentHeadingHTML = `<a class="nav-link py-1" href="#${key}">`
-        + `${this.navigableHeadings[key].text}&#x200E;</a>\n`;
+      // Add v-pre to prevent text interpolation for {{ }} wrapped in {% (end)raw %}
```

Comment:
Is this a typo on the `(end)raw` or is this intended?

Reply:
I used it to mean `{% raw %} {% endraw %}` 😂